src/raw/client.rs

```diff
@@ -256,7 +259,9 @@ impl Client {
                 requests::new_raw_scan_request(range, limit, self.key_only, self.cf.clone())
                     .execute(self.rpc.clone()),
             )
-        }
+        };
+
+        request.await
```

Comment:
I think this function can be further simplified by directly returning the error in the first case and using `...execute.await` in the second.